sdk/nodejs/types/output.ts

```diff
@@ -353,6 +353,7 @@ export namespace accesscontextmanager {
              */
             enableRestriction: boolean;
         }
+
```

Comment:
Puzzled by these file changing.. I reran the sdk build locally a couple of times and it seems repeatable with go 1.19. I don't understand how that could affect it though. I see the same in your PR for kubernetes too @lblackstone https://github.com/pulumi/pulumi-kubernetes/pull/2123/files#diff-5f52293a7ebfdbe25cfe109ca90083860c4bd9c742fd5fdefaa68302cf41620f.